library/systemd/src/lib/yast2/system_service.rb

```diff
@@ -205,14 +205,16 @@ def substate
     def current_start_mode
       return @current_start_mode unless @current_start_mode.nil?
 
-      @current_start_mode =
-        if service.enabled?
-          :on_boot
-        elsif socket&.enabled?
-          :on_demand
-        else
-          :manual
-        end
+      @current_start_mode = start_mode_from(service, socket, :enabled?)
+    end
+
+    # Determines whether the start mode is set to the default values
```

Comment:
Actually the method sets the default mode, doesn't it?

Reply:
No, it does not set but reads the value. But the description is wrong. Thanks for noticing it.